### PR TITLE
#33 - workflow for checking Dockerfile

### LINKED ISSUE #33: Add tests for Docker building
It would be nice to have some tests running in CI to check if Dockerfile is built properly.

AC:
* for every pull request for `main` there should be Github Action ran with Dockerfile build

### PR DESCRIPTION
**Pull request by kamilpiech97**
This should close #33.

### CODE REVIEW

File: .github/workflows/check-dockerfile.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
```
Comment (krzysztofrewak) [inline .github/workflows/check-dockerfile.yml:1]:
```suggestion
name: Test Dockerfile build
```

File: .github/workflows/check-dockerfile.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (krzysztofrewak) [inline .github/workflows/check-dockerfile.yml:11]:
```suggestion
  build:
    name: Test Dockerfile build
    runs-on: ubuntu-20.04
```

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

**Review by krzysztofrewak — COMMENTED** [review]
Could we test it also for build with xdebug enabled?

**Comment by mtracz** [comment]
> Could we test it also for build with xdebug enabled?

No sense for me.

File: .github/workflows/check-dockerfile.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Test Dockerfile build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: docker build . --file Dockerfile
```
Comment (mtracz) [inline .github/workflows/check-dockerfile.yml:16]: Is it possible to use "matrix" feature here and pass different build args for PHP_VERSION? E.g. `8.1,8-fpm-alpine` or `latest`?
```yaml
example_matrix:
    strategy:
      matrix:
        php_version: [8.1,8-fpm-alpine, latest]
```

```shell
docker build --build-arg PHP_VERSION=${{ matrix.php_version }} . --file Dockerfile
```
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations

**Review by mtracz — CHANGES_REQUESTED** [review]

File: .github/workflows/check-dockerfile.yml (hunk 3 above)
Comment (kamilpiech97) [reply .github/workflows/check-dockerfile.yml:16]: I would stay with my solution :)

**Review by mtracz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**